_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -26,16 +26,21 @@ public static function get_player_data( $feed ) {
 		$player_data   = get_transient( $transient_key );
 
 		// Fetch data if we don't have any cached.
-		if ( false === $player_data ) {
+		if ( false === $player_data || ( defined( 'WP_DEBUG' ) && WP_DEBUG ) ) {
```

Comment:
We can pull this into a separate PR if you'd like.

Reply:
no problemo!